bacon/model/StorageCouchBaseDB.swift

```diff
@@ -172,6 +223,11 @@ class StorageCouchBaseDB {
                 let transactionData = try JSONSerialization.data(withJSONObject: transactionDictionary, options: [])
                 let currentTransaction = try JSONDecoder().decode(Transaction.self, from: transactionData)
                 transactions.append(currentTransaction)
+                // Retrieve and store the mapping of transaction to its id in database
+                let transactionDatabaseId = result.string(forKey: "id")
+                if transactionMapping[currentTransaction] == nil {
+                    transactionMapping[currentTransaction] = transactionDatabaseId
+                }
```

Comment:
When it's not nil, should it also be overwritten? I don't know how edit is going to be implemented but might be relevant there